ndk.rosinstall

```diff
@@ -471,62 +474,71 @@
+- tar:
+    local-name: resource_retriever
+    uri: https://github.com/ros-gbp/resource_retriever-release/archive/release/kinetic/resource_retriever/1.12.4-0.tar.gz
+    version: resource_retriever-release-release-kinetic-resource_retriever-1.12.4-0
+- tar:
+    local-name: robot_model
+    uri: https://github.com/ros-gbp/robot_model-release/archive/release/kinetic/robot_model/1.12.11-0.tar.gz
+    version: robot_model-release-release-kinetic-robot_model-1.12.11-0
 - tar:
     local-name: robot_state_publisher
     uri: https://github.com/ros-gbp/robot_state_publisher-release/archive/release/kinetic/robot_state_publisher/1.13.6-0.tar.gz
     version: robot_state_publisher-release-release-kinetic-robot_state_publisher-1.13.6-0
+# Not needed in Android.
 # - tar:
 #     local-name: ros/mk
 #     uri: https://github.com/ros-gbp/ros-release/archive/release/kinetic/mk/1.14.4-0.tar.gz
 #     version: ros-release-release-kinetic-mk-1.14.4-0
+# Metapackage, not needed.
 # - tar:
 #     local-name: ros/ros
 #     uri: https://github.com/ros-gbp/ros-release/archive/release/kinetic/ros/1.14.4-0.tar.gz
 #     version: ros-release-release-kinetic-ros-1.14.4-0
+# Only bash scripts, not needed.
 # - tar:
 #     local-name: ros/rosbash
 #     uri: https://github.com/ros-gbp/ros-release/archive/release/kinetic/rosbash/1.14.4-0.tar.gz
 #     version: ros-release-release-kinetic-rosbash-1.14.4-0
+# Scripts only, not needed.
 # - tar:
 #     local-name: ros/rosboost_cfg
 #     uri: https://github.com/ros-gbp/ros-release/archive/release/kinetic/rosboost_cfg/1.14.4-0.tar.gz
 #     version: ros-release-release-kinetic-rosboost_cfg-1.14.4-0
-- tar:
-    local-name: ros/rosbuild
-    uri: https://github.com/ros-gbp/ros-release/archive/release/kinetic/rosbuild/1.14.4-0.tar.gz
-    version: ros-release-release-kinetic-rosbuild-1.14.4-0
-- tar:
-    local-name: ros/rosclean
-    uri: https://github.com/ros-gbp/ros-release/archive/release/kinetic/rosclean/1.14.4-0.tar.gz
-    version: ros-release-release-kinetic-rosclean-1.14.4-0
+# Scripts only, not needed.
+# - tar:
+#     local-name: ros/rosbuild
+#     uri: https://github.com/ros-gbp/ros-release/archive/release/kinetic/rosbuild/1.14.4-0.tar.gz
+#     version: ros-release-release-kinetic-rosbuild-1.14.4-0
+# Scripts only, not needed.
+# - tar:
+#     local-name: ros/rosclean
+#     uri: https://github.com/ros-gbp/ros-release/archive/release/kinetic/rosclean/1.14.4-0.tar.gz
+#     version: ros-release-release-kinetic-rosclean-1.14.4-0
+# Scripts only, not needed.
 # - tar:
 #     local-name: ros/roscreate
 #     uri: https://github.com/ros-gbp/ros-release/archive/release/kinetic/roscreate/1.14.4-0.tar.gz
 #     version: ros-release-release-kinetic-roscreate-1.14.4-0
-# - tar:
-#     local-name: ros/roslang
-#     uri: https://github.com/ros-gbp/ros-release/archive/release/kinetic/roslang/1.14.4-0.tar.gz
-#     version: ros-release-release-kinetic-roslang-1.14.4-0
+- tar:
```

Comment:
I thing this isn't needed

Reply:
I have some local changes I haven't pushed; I tried also removing things like `rospy` (which shouldn't be needed as they are Python only) but I found some packages that use them as build dependencies so the build failed.
I'll push the changes today with a list "as clean as it can be" (i.e. removing as many packages as I can, but without adding extra patches for now).